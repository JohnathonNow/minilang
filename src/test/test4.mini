var L := [1, 2, "hello", 4, 5]

for Key, Value in L do
	print('Key = {Key}, Value = {Value}\n')
end

var M := {"a" is 100, 10 is "b"}

for Key, Value in M do
	print('Key = {Key}, Value = {Value}\n')
end

fun test(X) do
	let (A, B, C) := X
	print('A = {A}, B = {B}, C = {C}\n')
end

test((1, 2, 3))
test([1, 2, 3])