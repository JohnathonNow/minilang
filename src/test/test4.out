Key = 1, Value = 1
Key = 2, Value = 2
Key = 3, Value = hello
Key = 4, Value = 4
Key = 5, Value = 5
Key = a, Value = 100
Key = 10, Value = b
